connector/src/main/java/org/geysermc/connector/entity/living/animal/horse/AbstractHorseEntity.java

```diff
@@ -47,6 +51,18 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
             metadata.getFlags().setFlag(EntityFlag.SADDLED, (xd & 0x04) == 0x04);
             metadata.getFlags().setFlag(EntityFlag.EATING, (xd & 0x10) == 0x10);
             metadata.getFlags().setFlag(EntityFlag.STANDING, (xd & 0x20) == 0x20);
+
+            // Bedrock uses DISPLAY_ITEM to work out if the mouth is open
+            metadata.put(EntityData.DISPLAY_ITEM, (xd & 0x40) == 0x40 ? 128 : 0);
```

Comment:
Is 128 supposed to be the wheat ID?

Reply:
Seems to be a magic number, maybe its time or something. Trying with 2 different foods for the horse gives the same number.

Reply:
If we can't work out what it is, then put a note saying that.